### PR TITLE
[feat] [RT] FIPS Self-test  (KATs only)

### PR DESCRIPTION
**Pull request by rusty1968**
This commit implements a MB FIPS command which perform a subset of the required FIPS Self-Tests (KAT tests).

### CODE REVIEW

**Comment by rusty1968** [comment]
https://github.com/chipsalliance/caliptra-sw/issues/321

**Review by jhand2 — APPROVED** [review]

File: runtime/src/fips.rs [hunk 1]
```diff
@@ -61,4 +64,29 @@ impl FipsModule {
     fn zeroize(env: &mut Drivers) {
         env.regions.zeroize();
     }
+
+    fn execute_kat(env: &mut Drivers) -> CaliptraResult<()> {
```
[the remark this answers is not in the document]
Comment (rusty1968) [reply runtime/src/fips.rs:68]: Is it ok, if I follow up in the next commit with the authentication change?

[the remark this answers is not in the document]
Comment (rusty1968) [reply runtime/src/fips.rs]: Done

**Comment by rusty1968** [comment]
@jhand2 , @korran  : Added a commit to remove LMS and SHA-1 until we confirm with John there is no issue with adding firmware based crypto algorithms in RT KAT.

**Review by jhand2 — DISMISSED** [review]

**Review by jhand2 — APPROVED** [review]

**Pull request merged**